plugins/lite3_db/Makefile

```diff
@@ -0,0 +1,24 @@
+
+PATH := $(PATH):/home/harvey/links/deriving
```

Comment:
Absolute path dependency, maybe this should be ../../deriving or something?

Reply:
I think it should be `PATH := $(PATH):deriving`, the deriving preprocessor is in a direct subdirectory called "deriving" relative to the Makefile.